<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportStats</class>
 <widget class="QDialog" name="ExportStats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qute Stats Export</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="statsText">
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveStatsButton">
       <property name="text">
        <string>Save to File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyStatsButton">
       <property name="text">
        <string>Copy to Clipboard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeStatsButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
